mldsa/config.h

```diff
@@ -43,13 +43,83 @@
  *
  * Description: The prefix to use to namespace global symbols from mldsa/.
  *
+ *              In a multi-level build (that is, if either
+ *              - MLD_CONFIG_MULTILEVEL_WITH_SHARED, or
+ *              - MLD_CONFIG_MULTILEVEL_NO_SHARED,
+ *              are set, level-dependent symbols will additionally be prefixed
+ *              with the parameter set (44/65/87).
+ *
  *              This can also be set using CFLAGS.
  *
  *****************************************************************************/
 #if !defined(MLD_CONFIG_NAMESPACE_PREFIX)
 #define MLD_CONFIG_NAMESPACE_PREFIX MLD_DEFAULT_NAMESPACE_PREFIX
 #endif
 
+/******************************************************************************
+ * Name:        MLD_CONFIG_MULTILEVEL_WITH_SHARED
+ *
+ * Description: This is for multi-level builds of mldsa-native only. If you
+ *              need only a single parameter set, keep this unset.
+ *
+ *              If this is set, all MLD_CONFIG_PARAMETER_SET-independent
+ *              code will be included in the build, including code needed only
+ *              for other parameter sets.
+ *
+ *              Example: mld_shake256_init is shared across all parameter sets.
+ *              If this option is set for a build with MLD_CONFIG_PARAMETER_SET
+ *              == 65, it would be included even though it's also needed for
+ *              levels 44 and 87.
```

Comment:
This is a bad example as `mld_shake256_init` is also needed for 65.
A better example would be `mld_poly_uniform_eta_4x` - we will need to versions of that: one for eta=2 and one for eta=4. 65 only need the eta=4 version, but the shared build would include both. But i am not sure if you have implemented that yet.

Reply:
aha yup, silly mistake! Thank you -- fixed